packages/react/src/ErrorBoundaryGroup.tsx

```diff
@@ -18,22 +18,24 @@ export type ErrorBoundaryGroupProps = PropsWithChildren<{
   blockOutside?: boolean
 }>
 
+const increase = (prev: number) => prev + 1
+
 /**
  * ErrorBoundaryGroup is Component to manage multiple ErrorBoundaries
  * @see {@link https://suspensive.org/docs/react/ErrorBoundaryGroup}
  */
 export const ErrorBoundaryGroup = ({ blockOutside = false, children }: ErrorBoundaryGroupProps) => {
-  const [resetKey, reset] = useKey()
+  const [resetKey, reset] = useReducer(increase, 0)
```

Comment:
Memory will be better because we just use `increase` function every time.
`useReducer with `increase` is more intuitive than `useKey`. 👍

Reply:
COOL!! We didn't have to make our own new hook! That's so cool